src/audio/multiband_drc/multiband_drc.c

```diff
@@ -328,7 +336,10 @@ static int multiband_drc_process(struct processing_module *mod,
 		}
 	}
 
-	cd->multiband_drc_func(mod, source, sink, frames);
+	if (cd->process_enabled)
```

Comment:
would it be possible to reassign `cd->multiband_drc_func` in `multiband_drc_process_enable()` and avoid the `if` here?